examples/tiny-kafka/README.md

```diff
@@ -1,16 +1,157 @@
+>
+> Docker isn't running or you're otherwise not able to talk to it. Check your Docker installation, then try again.
+
+&nbsp;
+
+
+## Package it
+
+
+Zarf is (at heart) a tool for making it easy to get software from _where you have it_ to _**where you need it**_&mdash;specifically, across an airgap. Since moving bits is so core to Zarf the idea of a "ready-to-move group of software" has a specific name&mdash;the _package_.
+
+All of the software a Zarf cluster runs is installed via package&mdash;for many reasons like versioning, auditability, etc&mdash;which means that if you want to run a  in your cluster you're going to have to build a package for it.
+
+Luckily, this is very easy to do&mdash;package contents are defined by simple, declarative yaml files and _we've already made one for you_. To build this package you simply:
+
+```sh
+cd <zarf dir>/examples/tiny-kafka     # directory with zarf.yaml, and
+zarf package create --confirm         # make the package
+```
+
+Watch the terminal scroll for a while. Once things are downloaded & zipped up and you'll see a file ending in `.tar.zst` drop.  _That's_ your package.
+
+&nbsp;
+
+
+## Deploy it
+
+
+It's time to feed the package you built into your cluster.
+
+Since you're running a Zarf cluster directly on your local machine&mdash;where this package & `zarf` binary _already are_&mdash;deploying the package is very simple:
+
+```sh
+zarf package deploy zarf-package-kafka-strimzi-demo.tar.zst --confirm
+```
+
+In a couple seconds the cluster will have loaded your package.
+
+&nbsp;
+
+## Use it
 
 Testing will require JDK and the kafka tools:  `sudo apt install openjdk-14-jdk-headless` for Ubuntu.  More details can be found at https://kafka.apache.org/quickstart.  Steps to test:
 
-1. Install JDK and extract the Kafka tools from the package `/opt/kafka.tgz`
+1. Install JDK and extract the Kafka tools from the package `kafka.tgz`
 2. Get the Nodeport: `NODEPORT=$(kubectl get service demo-kafka-external-bootstrap -n kafka-demo -o=jsonpath='{.spec.ports[0].nodePort}{"\n"}')`
 3. For pub: `./bin/kafka-console-producer.sh --broker-list localhost:$NODEPORT --topic cool-topic`
 4. For sub: `./bin/kafka-console-consumer.sh --bootstrap-server localhost:$NODEPORT --topic cool-topic`
+
+&nbsp;
+
+## Cleanup
+
+
+Once you've had your fun it's time to clean up.
+
+In this case, since the Zarf cluster was installed specifically (and _only_) to serve this example, clean up is really easy&mdash;you just tear down the entire cluster:
+
+```sh
+kind delete cluster
+```
+
+It only takes a couple moments for the _entire cluster_ to disappear&mdash;long-running system services and all&mdash;leaving your machine ready for the next adventure.
+
+&nbsp;
+
+---
+
+&nbsp;
+
+### Credits
+
+✨ Special thanks to these fine references! ✨
+
+- https://www.reddit.com/r/programming/comments/nap4pt/dos_gaming_in_docker/
+
+- https://earthly.dev/blog/dos-gaming-in-docker/
```

Comment:
don't need this stuff, these refs are for the game example

Reply:
haha yeah I had already removed it, just pushed it up.

Reply:
too much copy/paste fun